vscode-typescript/track.yml

```diff
@@ -0,0 +1,50 @@
+slug: vscode-typescript
+id: 0dxngfkws4op
+type: track
+title: VSCode & TypeScript
+teaser: Edit and test TypeScript applications in VSCode
+description: |-
+  A basic track for editing a TypeScript application and running tests for it.
+  The track checks are failing based on the test results.
+
+  ## Using this template
+  To create your own track from this template, run this command:
+  ```
+  instruqt track create \
+      --from templates/container \
+      --title "My VSCode & TypeScript track"
+  ```
+icon: https://cdn.instruqt.com/assets/templates/ts.png
+tags: []
+owner: templates
+developers:
+- borodean@gmail.com
```

Comment:
Can you change this to the Instruqt account.

Reply:
Yes this is probably a good idea. Not sure if we have an instruqt account, we should also make this consistent on the other templates.

Reply:
Updated in https://github.com/instruqt/templates/pull/4.